aion/web/views/session_view.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Aion.SessionView do
+  use Aion.Web, :view
+
+  def render("error.json", %{}) do
+    %{"data": "failed to authorise"}
+  end
+
+  def render("login.json", %{jwt: jwt}) do
```

Comment:
The `jwt` cannot be changed to `token` cause it would require a change in Guardian's implementation, right?